cabal-install/src/Distribution/Client/VCS.hs

```diff
@@ -305,7 +314,43 @@ vcsDarcs =
 
     vcsSyncRepos :: Verbosity -> ConfiguredProgram
                  -> [(SourceRepositoryPackage f, FilePath)] -> IO [MonitorFilePath]
-    vcsSyncRepos _v _p _rs = fail "sync repo not yet supported for darcs"
+    vcsSyncRepos verbosity prog = \ case
+      [] -> pure []
```

Comment:
```suggestion
      [] -> return []
```

Though I like `pure` too, in both this file and the rest of the repo `return` is used when in a monadic context. (same for the other `pure`s)

Reply:
Done